invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsField.js

```diff
@@ -107,21 +110,23 @@ class CreatibutorsFieldForm extends Component {
               );
             })}
           </List>
-          <CreatibutorsModal
-            onCreatibutorChange={this.handleOnContributorChange}
-            action="add"
-            addLabel={modal.addLabel}
-            editLabel={modal.editLabel}
-            roleOptions={sortOptions(roleOptions)}
-            schema={schema}
-            autocompleteNames={autocompleteNames}
-            trigger={
-              <Button type="button" icon labelPosition="left">
-                <Icon name="add" />
-                {addButtonLabel}
-              </Button>
-            }
-          />
+          <OverridableContext.Provider value={overriddenComponents}>
```

Comment:
By putting the context provider here, we are risking having many nested context providers and it might have unpredicted consequences (one of them being confusion). Why not having the context provider at the level of the form rendering (RDMDepositForm in invenio-app-rdm?)

Reply:
@mkloeppe Could you look into this?

Reply:
I removed the Context Provider. Indeed there already is a context provider in deposit form's index.js